ui/components/AutomationsTable.tsx

```diff
@@ -115,13 +115,11 @@ function AutomationsTable({ className, automations, hideSource }: Props) {
   if (hideSource) fields = _.filter(fields, (f) => f.label !== "Source");
 
   return (
-    <div className={className}>
```

Comment:
We lost this `className`, so our component is no longer a `styled-component`. You can pass `className` straight to `FilterableTable` if you want.